aggregator.py

```diff
@@ -213,12 +215,14 @@ def flush(self, timestamp, interval):
 class Histogram(Metric):
     """ A metric to track the distribution of a set of values. """
 
-    def __init__(self, formatter, name, tags, hostname, device_name):
+    def __init__(self, formatter, name, tags, hostname, device_name,
+            aggregates=None, percentiles=None):
         self.formatter = formatter
         self.name = name
         self.count = 0
         self.samples = []
-        self.percentiles = [0.95]
+        self.aggregates = aggregates or []
+        self.percentiles = percentiles or [0.95]
```

Comment:
probably a edge case but because of this, if you don't want to collect percentiles at all, it wouldn't work.